libr/core/disasm.c

```diff
@@ -2993,11 +2993,11 @@ static char * resolve_fcn_name(RAnal *anal, const char * func_name) {
 	return r_anal_type_func_guess (anal, (char*)func_name);
 }
 
-static bool can_emulate_metadata(RAnal * a, ut64 at) {
+static bool can_emulate_metadata(RCore * core, ut64 at) {
```

Comment:
handle asm.emuwrite=? to not change the value (return false) and show the help with all the possible values